src/test/java/org/kohsuke/github/AppTest.java

```diff
@@ -291,6 +291,42 @@ public void testGetIssues() throws Exception {
         String readRepoString = GitHub.getMappingObjectWriter().writeValueAsString(closedIssues.get(0));
     }
 
+    @Test
+    public void testQueryIssues() throws IOException {
```

Comment:
These queries offer a pretty good range of coverage.  
You have one without a call to `label()` and one with a single call, but we also need one with two calls to label
We also need passing `label(null)` and `label("")`.  It's a pain but there's at least that many possible scenarios that need to be checked.